### ISSUE TITLE
Test the SmallRye Health configuration 

### ISSUE THREAD

**Issue by antoinesd** [quarkusio/quarkus#7031]
**Description**
Test that configuration changes are taken into account

**Comment by paulrobinson** [comment]
Thanks for opening this. Do you know if this is already covered by the MP TCK tests? We've now got agreement that we don't need to duplicate tests in Quarkus that are already covered by the MP TCK.

**Comment by jmartisk** [comment]
I don't think this is covered in TCK, because Quarkus configuration mechanisms are inherently Quarkus-specific. AFAIK the only two things that you can configure now are paths to the health check endpoints and a flag whether health checks registered by other extensions should be enabled.
@antoinesd If you wish I can try to come up with some tests for the endpoint paths. Or is there anything else that you feel needs a test?

**Comment by geoand** [comment]
Is there anything that needs to be done for this or can we close it?
